xml/System.Runtime.InteropServices/MemoryMarshal.xml

```diff
@@ -357,20 +447,28 @@
         <Parameter Name="span" Type="System.Span&lt;T&gt;" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="span">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="T">The type of items in the span.</typeparam>
+        <param name="span">The span from which the reference is retrieved.</param>
+        <summary>Returns a reference to the element of the span at index 0.</summary>
+        <returns>A reference to the element at index 0. </returns>
+        <remarks> 
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+If the span is empty, this method returns a reference to the location where the element at index 0 would have been stored. Such a reference may or may not be `null`. The returned reference can be used for pinning, but it must never be dereferenced.
+
+       ]]></format> 
+        </remarks>        
       </Docs>
     </Member>
     <Member MemberName="Read&lt;T&gt;">
-      <MemberSignature Language="C#" Value="public static T Read&lt;T&gt; (ReadOnlySpan&lt;byte&gt; source) where T : struct;" />
+      <MemberSignature Language="C#" Value="public static T Read&lt;T&gt; (ReadOnlySpan&lt;Byte&gt; source) where T : struct;" />
```

Comment:
same comment as earlier on `Byte` vs. `byte`.

Reply:
This seems to have been a copy and paste error -- I wanted to change byte to Byte in 2 places, and changed them also in C# and F# syntax. I've corrected them.